libvast/include/vast/system/actors.hpp

```diff
@@ -263,19 +247,15 @@ using index_actor = typed_actor_fwd<
   caf::reacts_to<atom::subscribe, atom::create,
                  partition_creation_listener_actor, send_initial_dbstate>,
   // Evaluates a query, ie. sends matching events to the caller.
+  caf::replies_to<atom::evaluate, query, uint64_t>::with<query_response>,
```

Comment:
Can you document what the additional parameter is for?

Reply:
This handler has been removed.